[IMP] empty

arpi-odoo opened the pull request.

arpi-odoo committed:
[FIX] hr: new contract button invisible

arpi-odoo committed:
[IMP] hr_skills: cleanup certification reporting

arpi-odoo committed:
[FIX] hr_holidays: fix traceback leave type

arpi-odoo committed:
[FIX] hr: fix delete version

arpi-odoo committed:
[FIX] hr: fix various bugs

mepe-odoo committed:
[FIX] hr_holidays: fix accrual day selection

arpi-odoo committed:
[IMP] empty

robodoo commented:
This PR targets the un-managed branch odoo-dev/odoo:master-hr-onboarding, it needs to be retargeted before it can be merged.

ibmah-odoo committed:
[FIX] hr: hide "New Contract" button when no start date is set

When there's no start date for a contract of an employee, it doesn't
make sense to create a contract for them, so this ensures that the new
contract button is only visible when a start date has been set.

task-5446065

ibmah-odoo committed:
[FIX] hr_holidays: fix time-off traceback

This fixes a traceback raised by trying to create a timeoff using the
smart buttons on time-off types

Steps to reproduce: Time Off >> Configuration >> Time Off Types >> Open
any type, navigate to the Smart button Time Off >> Click on New >> Time
Off Type (Leads to an error.)

Bug cause: Missing second argument of the comparison operation in the
search function

Solution: Add the value entered by the user to the comparison with
remaining virtual leaves operator function

task-5447241

arpi-odoo committed:
[FIX] hr_skills: fix certification report
